code/modules/surgery/emergency_cardioversion_recovery.dm

```diff
@@ -0,0 +1,119 @@
+/datum/surgery/cardioversion
```

Comment:
By the by, we already have one revival surgery datum: `/datum/surgery/advanced/revival`.
Any plan in merging both together or a reason this should exist as a separate surgery?

Reply:
Revival I believe works for any and all cases, whereas this will revive those within the defib window and is limited by normal defib restrictions. Though, I should give revival a pass over to make sure they don't die instantly from organ damage.

Reply:
Also I reworked it so it's a ghetto defib method now. Not got a defib? Use a battery applied directly to their heart by cutting open their chest!